spec/filters/integration/jdbc_static_spec.rb

```diff
@@ -83,6 +83,16 @@ module LogStash module Filters
         end
       end
 
+      context "under normal conditions with prepared statement" do
+        let(:lookup_statement) { "SELECT * FROM servers WHERE ip LIKE ?" }
+        it "enhances an event" do
+          settings.local_lookups.prepared_parameters = [parameters_rhs]
```

Comment:
This currently fails testing - https://travis-ci.org/logstash-plugins/logstash-filter-jdbc_static/builds/617288523

To replace the value of `parameters` with `prepared_parameters` for this test case, I think you will need to replace the whole `local_lookups` block in `:settings`, rather than just the lookup statement: you will need to remove the existing `parameters` setting as well as include a new `prepared_parameters` setting.

Reply:
Done :-) thanks @robbavey now Travis is green